src/collective/js/cycle2/upgrades/profiles/default/browserlayer.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
```

Comment:
@rodfersou can we remove the layer programmatically? that way we have no need of registering a profile